server/src/api.py

```diff
@@ -99,6 +99,7 @@ def login() -> LoginResponse:
         error, new_user = User.add_google_user(googleToken)
         if not error:
             login_user(new_user, remember=True)
+            get_people_connections_list(googleToken)
```

Comment:
This is fine for local testing, but probably don't want to merge in this behavior.

Reply:
tru